content/scalers/azure-storage-blob.md

```diff
@@ -0,0 +1,73 @@
++++
+fragment = "content"
+weight = 100
+title = "Azure Storage Blob"
+background = "light"
++++
+
+Scale applications based on Azure Storage Blobs.
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `azure-blob` trigger for Azure Storage Blob.
+
+```yaml
+triggers:
+  - type: azure-blob
+    metadata:
+      containerName: functions-blob # Required: Container name
```

Comment:
I presume this is aimed at the storage container, not runtime container?

Might be better to clarify this:
```suggestion
      containerName: functions-blob # Required: Name of Azure Blob Storage container
```

Or even rename it to `blobContainerName`?

Reply:
yeah. better rename it to "blobContainerName" for the clarity. I'll update the code and then update this doc.

Reply:
I've updated the code and rename the param to `blobContainerName`. Also updated the docs. Please review again.

Reply:
Thanks!